test/message/esm_loader_not_found_cjs_hint.out

```diff
@@ -0,0 +1,18 @@
+(node:*) ExperimentalWarning: The ESM module loader is experimental.
+(node:*) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
+internal/modules/esm/resolve.js:*
+    url = moduleWrapResolve(specifier, parentURL);
+          ^
+
+Error: Cannot find module * imported from *, the resolved CommonJS module would have been: *
```

Comment:
The `*` seems somewhat confusing. Should this not be an absolute or relative path?

Reply:
Yes they would be both absolute paths. I should improve this test.

Reply:
Maybe something like this would be better?
```
Error: Cannot find module */test/common/fixtures imported from */node/, the resolved CommonJS module would have been: */test/common/fixtures.js
```